style.css

```diff
@@ -0,0 +1,60 @@
+body {
+    background-color: #efece7;
+}
+header , section , footer {
+	margin: 0 auto;
+	width: 1200px;
+
+}
+aside, article {
+	display: inline-block;
+	width: 49%;
+}
+h1 {
+	font-size: 83px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Magazine_page1/pullRequest?prid=5459324)